toolkit/checks.nu

```diff
@@ -97,6 +97,81 @@ export def test [
     }
 }
 
+# check that all the tests pass using a fancy nextest
+#
+# Saves a junit output file in ./target/nextest/
+export def --wrapped "test ci" [
+    --features: list<string> = [] # the list of features to run the tests on
+    --workspace # run the command on the whole workspace
+    --profile: string = "ci" # Nextest profile to use
+    --cargo-profile: string # Cargo profile to use
+    --config: path # Path to the nextest config
+    --groups # Show the grouped tests
+    ...rest # Args to pass directly to `nextest run`
+] {
+    use std/log
+    let toplevel: path = match (which jj git).0? {
+        null => "."
+        _ => (
+            try {
+                ^jj workspace root | complete
+            } catch {
+                ^git rev-parse --show-toplevel | complete
+            }
+        ).stdout
+    }
+    let config: path = $config
+    | default ($toplevel | path join .cargo nextest.toml)
```

Comment:
This formatting isn't correct with lines beginning with pipes. See if topiary-nushell will format it better.

Reply:
Formatted it like topiary would.

Reply:
Yup, indent like your example